app/activeplan/homepage/get.controller.js

```diff
@@ -31,6 +31,14 @@ const getHomepage = async (req, res) => {
     // objectives default to active if not caught with the rules below
     currentObjective.type = 'active'
 
+    currentObjective.actionsCompleted = currentObjective.actions.reduce((total, { status }) => {
+      let currentTotal = total
+      if (['COMPLETED', 'PARTIALLY_COMPLETED', 'ABANDONED'].includes(status)) {
+        currentTotal += 1
+      }
+      return currentTotal
```

Comment:
This feels more like a filter:
currentObjective.actionsCompleted = currentObjective.actions.filter(({ status }) => 
['COMPLETED', 'PARTIALLY_COMPLETED', 'ABANDONED'].includes(status)).length